docs/markdown/snippets/eval-unary-op-on-none.md

```diff
@@ -0,0 +1,4 @@
+## Fix unhandled exception when applying unary operator to None value
+
+Writing `-subdir('a')` or `not subdir('a')` now raises `InvalidCode` exception
+instead of dumping a stacktrace.
```

Comment:
I don't think this needs a release notes snippet, we don't usually include bug fixes and that never worked, we just now report proper error.